### PR TITLE
improving useful addresses section

### PR DESCRIPTION
**Pull request by githubdoramon**

### CODE REVIEW

File: docs/dev/building-on-zksync/useful-address.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ Here are some contract addresses that may be useful and/or of interest to those
 | l1_allow_list_addr            | [0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA](https://etherscan.io/address/0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA#code)|
 
 
-## Root hashes
```
Comment (vladbochok) [inline docs/dev/building-on-zksync/useful-address.md:21]: Why we have this? It is not related to the user and hard to maintain, probably just remove the section?

File: docs/dev/building-on-zksync/useful-address.md [hunk 2]
```diff
@@ -90,9 +90,9 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
...
+| COMBO                         | Furucombo     | [0xffffffff2ba8f66d4e51811c5190992176930278](https://etherscan.io/address/0xffffffff2ba8f66d4e51811c5190992176930278)|[0xc2B13Bb90E33F1E191b8aA8F44Ce11534D5698E3](https://explorer.zksync.io/address/0xc2B13Bb90E33F1E191b8aA8F44Ce11534D5698E3#contract)|
+| PERP                          | Perpetual     |  [0xbC396689893D065F41bc2C6EcbeE5e0085233447](https://etherscan.io/address/0xbC396689893D065F41bc2C6EcbeE5e0085233447)|[0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601](https://explorer.zksync.io/address/0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601#contract)|
+| USDC                          | USD Coin      | [0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48)|[0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4](https://explorer.zksync.io/address/0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4#contract)|
```
Comment (krofax) [inline docs/dev/building-on-zksync/useful-address.md:98]: @githubdoramon Isn't this suppose to be Circle, as the name?

Comment (githubdoramon) [reply docs/dev/building-on-zksync/useful-address.md:98]: Circle is the company... the token is USD Coin

File: docs/dev/building-on-zksync/useful-address.md [hunk 3]
```diff
@@ -90,9 +90,9 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
 
 ## Known tokens
```
[the remark this answers is not in the document]
Comment (githubdoramon) [reply docs/dev/building-on-zksync/useful-address.md:91]: Good point... Maybe "Mainnet known tokens"?

File: docs/dev/building-on-zksync/useful-address.md [hunk 4]
```diff
@@ -51,7 +51,7 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
 ## Testnet contract addresses
 
 
-|      Contract Name            | Address                                                                                                                          | 
+|      Contract                 | Address                                                                                                                          | 
 | ----------------------------- | ---------------------------------------------------------------------------------------------------------------------------------|
 | diamond_cut_facet_addr        | [0x6F883c7DA8Ec1918fa83C5E57F239f47f03b135d](https://goerli.etherscan.io/address/0x6F883c7DA8Ec1918fa83C5E57F239f47f03b135d#code)|
```
[the remark this answers is not in the document]
Comment (githubdoramon) [reply docs/dev/building-on-zksync/useful-address.md]: thought that this was something that remains "forever"... agree, lets remove then

File: docs/dev/building-on-zksync/useful-address.md (hunk 1 above)
Comment (githubdoramon) [reply docs/dev/building-on-zksync/useful-address.md:21]: thought that this was something that remains "forever"... agree, lets remove then

File: docs/dev/building-on-zksync/useful-address.md [hunk 5]
```diff
@@ -62,33 +51,21 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
...
-
-|          Name                 | Hash                                                               | 
-| ----------------------------- | -------------------------------------------------------------------|
-| vk_commitment_basic_circuits  | 0x142a364ef2073132eaf07aa7f3d8495065be5b92a2dc14fda09b4216affed9c0 |
-| vk_commitment_leaf            | 0x101e08b00193e529145ee09823378ef51a3bc8966504064f1f6ba3f1ba863210 |
-| vk_commitment_node            | 0x1186ec268d49f1905f8d9c1e9d39fc33e98c74f91d91a21b8f7ef78bd09a8db8 |
-| genesis_block_commitment      | 0x60d946bb425a7a5f6104fe7ef4b854fac4c32b6ba5b3601c82014fa28f6511d6 |
-| genesis_root                  | 0xc4b302452a77d72c6ad618ab1f18ec3a7699520dc053e767eca6427d24bd0848 |
-| genesis_tx_hash               | 0xb99ebfea46cbe05a21cd80fe5597d97b204befc52a16303f579c607dc1ac2e2e |
+| L1AllowList                   | [0xCbA757b4f0527b535bE80720325064058FC4A306](https://goerli.etherscan.io/address/0xCbA757b4f0527b535bE80720325064058FC4A306#code)|
```
Comment (vladbochok) [inline docs/dev/building-on-zksync/useful-address.md:54]:
```suggestion
| AllowList                   | [0xCbA757b4f0527b535bE80720325064058FC4A306](https://goerli.etherscan.io/address/0xCbA757b4f0527b535bE80720325064058FC4A306#code)|
```

**Review by vladbochok — APPROVED** [review]
Looks good! just leave a nits

**Review by vladbochok — APPROVED** [review]

**Pull request merged**